stacks/dotnetcore31/devfile.yaml

```diff
@@ -35,7 +35,8 @@ components:
         - name: ASPNETCORE_URLS
           value: http://*:8080
       endpoints:
-        - name: http-dotnetcore
+        - name: https-dotnetcore
```

Comment:
trying to start dotnet devfile in che:
```
Failed to fetch resources. Devfile schema validation failed. Error: 0: instance.components[0].container.endpoints[0].name does not meet maximum length of 15
```
![screenshot-che-dogfooding apps che-dev x6e0 p1 openshiftapps com-2024 05 17-16_27_52](https://github.com/devfile/registry/assets/1271546/2252f6d3-babe-4734-a23b-2e60fcce1451)

the same error in the PR check build

Maybe replace the endpoint's name with **https-dotnet** or just **dotnetcore**

Reply:
done
**https-dotnet** used